Cargo.toml

```diff
@@ -2,8 +2,9 @@
 name = "cargo-ghp-upload"
 version = "0.1.0"
 authors = ["CAD97 <cad97@cad97.com>"]
+description = "Upload documentation straight to GitHub Pages, maintaining branch seperation and history"
 
-include = ["/src/", "/LICENSE-APACHE", "/LICENSE-MIT", "/README.md"]
+include = ["/src/**/*", "Cargo.toml", "/LICENSE-APACHE", "/LICENSE-MIT", "/README.md"]
```

Comment:
I don't think I've ever bothered or seen another project do this.

Reply:
Yeah, it's not really necessary since all git-ignored files aren't included in the package.

Somewhat of note, it means that I didn't include the `Cargo.lock`...